src/client/src/translations/whitelist_en.json

```diff
@@ -93,9 +92,9 @@
   "selectFrontendFramework.selectFrontendFramework",
   "pages.limitedPagesMessage",
   "pages.overlimitPagesMessage",
+  "pages.noPageGeneration",
   "pages.maxPagesText",
   "selectPages.pagesTitleQuestion",
-  "selectPages.webAppTitleQuestion",
```

Comment:
why did you remove this?

Reply:
I did not. I ran `npm run manage:translations` as well but I don't know why it still shows up. I don't know if it is being used anywhere.